app/views/admin/movies/index.html.erb

```diff
@@ -14,6 +14,8 @@
   </li>
 <% end %>
 
+<%= link_to('Actors', admin_actors_path) %>
```

Comment:
Look into adding header and footer, tomorrow you are going to be adding bootstrap.
note:
Always remember....... 'DRY'.